### PR TITLE
Pull ram selection

### PR DESCRIPTION
**Pull request by ghost**
User-wrapper ready for pull request.

### CODE REVIEW

File: projects/revaturecloud/selection/src/lib/services/selection.service.ts [hunk 1]
```diff
@@ -103,6 +103,26 @@ export class SelectionService {
       );
   }
 
+
+  ///
```
Comment (JGrippo) [inline projects/revaturecloud/selection/src/lib/services/selection.service.ts]: Delete this.

File: projects/revaturecloud/selection/src/lib/shared/search.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  pure: false
+})
+export class SearchPipe implements PipeTransform {
+
+  transform(value: any, searchString: string): any {
+    if(value.length === 0 || searchString === ''){
+      return value;
+    }
+    const foundUsers = [];
+    for(const user of value){
```
Comment (JGrippo) [inline projects/revaturecloud/selection/src/lib/shared/search.pipe.ts]: Ctrl k +ctrl f

Comment (JGrippo) [reply projects/revaturecloud/selection/src/lib/shared/search.pipe.ts]: format it

File: src/app/app.module.ts [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
+    AppComponent
   ],
   imports: [
     BrowserModule,
     HttpClientModule,
     FormsModule,
     HttpModule,
     RouterModule,
-    SelectionModule
+    SelectionModule,
```
Comment (JGrippo) [inline src/app/app.module.ts]: Fix this.

**Review by JGrippo — CHANGES_REQUESTED** [review]
Make changes.

File: package.json [hunk 4]
```diff
@@ -22,6 +22,7 @@
     "@angular/platform-browser": "6.0.3",
     "@angular/platform-browser-dynamic": "6.0.3",
     "@angular/router": "6.0.3",
+    "angular-in-memory-web-api": "^0.6.0",
```
Comment (Abyssopelagic) [inline package.json]: should not have

**Review by JGrippo — APPROVED** [review]
Good job!

**Pull request merged**